css/code.css

```diff
@@ -148,3 +148,23 @@
 .highlight .w {
   color: #bbbbbb;
 }
+
+h2>a.anchor, h3>a.anchor, h4>a.anchor, h5>a.anchor, h6>a.anchor {
+  display: block;
+  margin-left: -1.5ex;
+  position: absolute;
+  text-decoration: none !important;
+  visibility: hidden;
+  z-index: 2;
+  transition: visibility 1s;
```

Comment:
I did this to stop the visibility transition being applied before the user's pointer could reach the chain symbol.